Trap Quest.inform/Inform/Extensions/Dresses/Cheerleader Outfit.i7x

```diff
@@ -1,6 +1,6 @@
 Cheerleader Outfit by Dresses begins here.
 
-A cheerleader outfit is a kind of overdress. A cheerleader outfit is unique. A cheerleader outfit is usually short. Understand "cheer", "leader" as cheerleader outfit.
+A cheerleader outfit is a kind of overdress. A cheerleader outfit is unique. A cheerleader outfit is usually short. A cheerleader outfit is usually audibly jiggling. Understand "cheer", "leader" as cheerleader outfit.
```

Comment:
I had two ideas for a benefit: Enemies would have a higher chance of going for breasts and/or friendly followers would stick around longer. Though I have no idea on how either might be implemented. I don't think that charisma makes much sense with it aimed to be embarrassing, though I guess you could argue against my second benefit similarly. If you implement the first idea, then another smaller benefit might balance out the negatives, though I can't think of one right now. 

A condition that checks the breasts of the player would make it equal to the way it normally spawns. Perhaps adding in a size check during each time it transforms after the fact, it would prevent it from never getting attached. For the schoolgirl outfit I think the unlucky modifier might be a good balance since it's something easily attained. However the cheerleader class is both a rarer class and a derivative from the schoolgirl when you get pigtails, so I don't think it's necessary there.

Reply:
Friendly followers actually *will* stay around a bit longer on average with this effect, because they will be able to stalk the player almost flawlessly, and so they won't run the risk of going "hmm I've actually ended up too far away from the player and can't be bothered to keep seeking them out". But yes we could also alter how quickly their "friendly boredom" increases, or what level it needs to reach for them to lose interest... however I would argue that in most cases, friendly NPCs following you around is a bad thing. Since it means that if they see you doing something lewd or private - anything from undressing to using the toilet to masturbating - they will lose favour with you.

The only real time it's good to have someone follow you around is if they love you enough to fight with you.

Increased chance of NPCs going for breasts is easy to implement but arguably only a slight positive, since if too many NPCs titfuck you it's still pretty intense, what with the increased sensation and addiction and eventually orgasms. Also it raises the question of whether perhaps the clothing should become `bsounding` every turn an NPC fucks your breasts! 😄 

If what you're suggesting is that the chance of audible jiggling appearing increases with breast size, I love that and think it's a great idea.

It's not impossible to check after each transformation, too. A little fiddly but can be done. The best way to do it would be to use the `To compute post transformation effect of (C - a clothing):` function which fires after a clothing has been transformed.

So the full block would look something like:

```
To compute post transformation effect of (C - a schoolgirl outfit):
	if a random number between 1 and 20 < the largeness of breasts, now C is audible jiggles.
```

Reply:
Having the clothes become bsounding when a titfuck happens would be great. 

Charisma and seductive bonuses are better than my two suggestions with that explanation. I was also thinking a magic bonus but I forget if that would make it too easy to swap over into the magical girl class. 

With this check, should there also be "if largeness of breast < 2, {break statement}" to prevent those with flat chests from acquiring it?

Reply:
If we are making audible jiggles only appear on schoolgirls upon transformation, and with RNG, I think it's fine to leave it without a positive effect. However I also would be fine with it giving a +1 to charisma.

> With this check, should there also be "if largeness of breast < 2, {break statement}" to prevent those with flat chests from acquiring it?

I'm not sure what check you mean? If you mean in the post transformation effect thing, that's already taken care of since a random number between 1 and 20 cannot be lower than 1. If we want to avoid it be lower than 2, since AA cup boobies are probably too small to make any noise, we could just change it to a random number between 2 and 20.

Since we're resetting all `bsounding` flags at the end of the turn, the best time to add a `bsounding` flag during titfucks is probably very early on in the turn, i.e. before monsters have their turn. So I would do this:

```
An all time based rule (this is the audible jiggling from titfucks rule):
	if there is a male monster penetrating breasts:
		repeat with C running through worn audible jiggles clothing:
			now C is bsounding.
```